braidpool_spec.md

```diff
@@ -460,33 +675,73 @@ formed, fully signed and valid bitcoin transactions that can be broadcast. See
 [Payout Authorization](#payout-authorization) for considerations on how to
 sign/authorize this UHPO transaction.
 
+[kp]
+
+The UHPO transactions are signed and broadcast using the braidpool P2P
+network. In case of the pool shutting down, the miners will have to figure out a
+communication channel to broadcast their signature shares and reach agreement on
+spending the UHPO.
+
+This sounds like a problem.
+
+[kp]
+
```

Comment:
Correct.

If we fully sign the UHPO transaction before it is needed, anyone can broadcast it and it's a DOS vector. If we don't, we have to figure out how to sign it after pool failure. Those are the only choices and we'll have to figure it out...